### ISSUE TITLE
Invalid SSL certificate

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#1011]
https://zerocracy.com uses invalid SSL certificate:
>zerocracy.com uses an invalid security certificate.<br/>The certificate is only valid for the following names: *.github.com, github.com, *.github.io, github.io <br/>Error code: SSL_ERROR_BAD_CERT_DOMAIN

but https://www.zerocracy.com has valid sertificate.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@g4s8 fixed. Try again in 24 hours, please.

**Comment by g4s8** [comment]
@yegor256 yes, https://zerocracy.com now returns error `ERR_CONNECTION_TIMED_OUT`

**Comment by yegor256** [comment]
@g4s8 that's because of GitHub problems, let's try in a few days again.

**Comment by yegor256** [comment]
@g4s8 I believe it's fixed?

**Comment by g4s8** [comment]
@yegor256 no, it's still not working:
http://zerocracy.com redirects to http://www.zerocracy.com
https://www.zerocracy.com works fine
https://zerocracy.com can't be opened (The connection has timed out)
